packages/system/kubevirt-csi-node/values.yaml

```diff
@@ -1 +1,3 @@
 storageClass: replicated
+csiDriver:
+  image: ghcr.io/cozystack/cozystack/kubevirt-csi-driver:0.25.0@sha256:445c2727b04ac68595b43c988ff17b3d69a7b22b0644fde3b10c65b47a7bc036
```

Comment:
Could you also add this to `make update` target, please?

Reply:
Okay fixed that https://github.com/cozystack/cozystack/pull/1165/commits/d200017f74e5df5ab17d232cc201a614c1316717